### PR TITLE
Update manage-search-schema.md

### PR DESCRIPTION
**Pull request by sympmarc**
Added a new table with variants on `RefinableString`. I chose to more closely follow the format of the UI for the **Managed property characteristics** rather than the way the table above works, since the characteristics don't match the wording or the order in the UI.

AFAIK, these variants aren't documented anywhere else, as Bingling for "RefinableStringFirst" (for example) returns zero hits.

### CODE REVIEW

File: SharePoint/SharePointOnline/manage-search-schema.md [hunk 1]
```diff
@@ -387,7 +387,19 @@ The following table provides an overview of the default unused managed propertie
 | Integer                   | 50        | Queryable                                                | Int00 to Int49                           |
 | Integer                   | 50        | Multivalued, Queryable, Refinable, Sortable, Retrievable | RefinableInt00 to RefinableInt49         |
 | String                    | 200 | Multivalued, Queryable, Refinable, Sortable, Retrievable | RefinableString00 to RefinableString199  |
-    
+
+Additional variants of RefinableString also exist for specific use cases. 
+
+| **Count** | **Multi** | **Query** | **Search** | **Retrieve** | **Refine** | **Sort** | **Managed property name range** | **Difference from RefinableStringXX** |
+| :-------- | :- | :- | :- | :- | :- | :- |:------------------------------------------------------- | :--------------------------------------- |
+| 40 | Y | Y | - | Y | Y | Y | RefinableStringFirst00 to RefinableStringFirst39  | Mappings to crawled properties - Include content from the first crawled property that is not empty, based on the specified order |
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-search-schema.md:395]:
```suggestion
| 40 | Y | Y | - | Y | Y | Y | RefinableStringFirst00 to RefinableStringFirst39  | Mappings to crawled properties - include content from the first crawled property that is not empty, based on the specified order. |
```

File: SharePoint/SharePointOnline/manage-search-schema.md [hunk 2]
```diff
@@ -387,7 +387,19 @@ The following table provides an overview of the default unused managed propertie
...
+| 30 | Y | Y | - | - | Y | Y | RefinableStringInternal00 to RefinableStringInternal29 | Not Retrieve |
+| 10 | Y | Y | - | Y | Y | Y | RefinableStringLn00 to RefinableStringLn09 | TBD |
+| 10 | - | Y | - | Y | Y | Y | RefinableStringSingleInternal00 to RefinableStringSingleInternal09 | Not Multi |
+| 30 | - | Y | - | - | Y | Y | RefinableStringSingleInternal10 to RefinableStringSingleInternal39 |  Not Multi, not Retrieve - Note this is the 10-39 range |
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-search-schema.md:399]:
```suggestion
| 30 | - | Y | - | - | Y | Y | RefinableStringSingleInternal10 to RefinableStringSingleInternal39 |  Not Multi, not Retrieve - note that this is the 10-39 range. |
```

File: SharePoint/SharePointOnline/manage-search-schema.md [hunk 3]
```diff
@@ -387,7 +387,19 @@ The following table provides an overview of the default unused managed propertie
...
+| 50 | - | Y | - | Y | Y | Y | RefinableStringWbOff00 to RefinableStringWbOff49 | Not Multi |
+| 50 | Y | Y | - | Y | Y | Y | RefinableStringWbOffFirst00 to RefinableStringWbOffFirst49 | Mappings to crawled properties - Include content from the first crawled property that is not empty, based on the specified order |
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/manage-search-schema.md:401]:
```suggestion
| 50 | Y | Y | - | Y | Y | Y | RefinableStringWbOffFirst00 to RefinableStringWbOffFirst49 | Mappings to crawled properties - include content from the first crawled property that is not empty, based on the specified order. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sympmarc, kindly commit the suggested minor changes. Many thanks.

**Comment by sympmarc** [comment]
Done, @JohanFreelancer9. This test is copied directly from the UI. Not sure how I de-capitalized the "i".

image

Mentioning @wobba for awareness.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by sympmarc** [comment]
@JoanneHendrickson @JohanFreelancer9 - @wobba and I have discussed a couple more changes to the table in the [Default unused managed properties](https://docs.microsoft.com/en-us/sharepoint/manage-search-schema#default-unused-managed-properties) section. I'd like to add those before you publish the changes above.

We're going to improve the first table to better match the UI and combine the two tables into one. Some of the RefinableString variants - specifically those with Internal in their name - will be hidden, and thus should not be in this doc.

**Comment by wobba** [comment]
The changes by @sympmarc aligns with an internal discussion we had in the product group.

**Comment by sympmarc** [comment]
@wobba @JoanneHendrickson @JohanFreelancer9 - I've merged the two tables and added "footnotes" in a column on the right headed **Notes**. I'm not sure what the convention should be with this additional information, but it doesn't fit very well inline.

**Comment by sympmarc** [comment]
I switched from footnotes to asterisks after input from @tomresing and @daisyfell.

I think this is good to go from my end!

**Comment by pamgreen-msft** [comment]
Hmm, I can't seem to add the 'sign off' label... @SerdarSoysal or @MikePlumleyMSFT, sorry to bug but can you? A lot of do si do on this one. <=)

**Comment by tomresing** [comment]
@pamgreen-msft that label isn't used in the public repo. I think we'd just merge it, which I could do after one further clarification.
@sympmarc what order is this table sorted in? It seems like it's grouped by the first column, but then I'd probably expect it to be sorted by the second column, either ascending or descending by number within the group.

**Comment by sympmarc** [comment]
I left the original rows in the order they were in originally. I added the new rows at the bottom, and I believe they are sorted in the order they display in the UI.

**Pull request merged**